platforms/substrate/charts/vitalam-ipfs-node/templates/configmap.yaml

```diff
@@ -22,4 +22,3 @@ data:
   ipfsApiPort: {{ include "vitalam-ipfs.ipfsApiPort" . }}
   ipfsSwarmPort: {{ .Values.config.ipfsSwarmPort | quote }}
   dataVolumeSize: {{ .Values.storage.dataVolumeSize | quote }}
-  
```

Comment:
newline please.

Reply:
This is fine, the wrong newline was removed